src/components/PhotoFrame.tsx

```diff
@@ -65,6 +65,11 @@ const ListItem = styled.div`
     justify-content: center;
 `;
 
+const PlaceHolderDiv = styled.div`
```

Comment:
this is need to show a placeholder thumbnail when the scrolling and we don't load the actual thumbnail